### ISSUE TITLE
Pronoms compléments de verbe comme modificateurs

### ISSUE THREAD

**Issue by perrier54** [bguil/UD-French-discussion#22]
Cela concerne le pronom clitique _y_ et le pronom relatif ou interrogatif _où_, quand il sont utilisés comme modificateurs.
_J'y travaille.
Où travailles-tu ?_
La question est de savoir si nous mettons OBL:MOD ou ADVMOD.
Par analogie avec _je travaille là-bas_, j'aurais tendance à mettre ADVMOD.

**Comment by sylvainkahane** [comment]
Je pense que la distinction entre advmod et obl:mod est purement catégorielle (et donc sans intérêt). On a qu'à considérer que les ADV sont advmod et les PRON sont obl:mod.
La vraie question est de savoir si _y_ et _où_ sont ADV ou PRON. Pour _y_ je dirai PRON à cause d'emplois comme _j'y pense_. Et je vois pas d'intérêt à avoir deux catégories pour _y_. Pour _où_, on peut le mettre ADV, mais il s'agit d'une classe d'ADV qui ont plus ou moins des distributions de (pro)noms puisqu'on les trouvent après des prép: d'où, vers où, par où (Cf. Kahane 2010 pour le cas vraiment clair des noms/pronoms temporels.)

PS Travailler est un peu compliqué:
- j'y travaille = je travaille à ma thèse : obl:arg
- je travaille à la fac est ambigu (je suis universitaire vs aujourdh'ui je travaille pas chez moi)

**Comment by perrier54** [comment]
Attention : il faut annoter ADVMOD uniquement quand les mots sont des modificateurs. Pour les exemples _je suis là_ ou _la ville où je suis_ il faut annoter OBL:ARG

**Comment by alineetienne** [comment]
Oui, c'est noté ! Je ferai attention à cette distinction lors de la vérification de l'annotation.

**Comment by sylvainkahane** [comment]
et il faudra convertir les ADV qui sont iobj en obl:arg dans les autres corpus
